src/main/resources/data/crimson/tags/items/logs.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
2 spaces in json

Reply:
Fixed in 500f65b0a3f45983d78cbaa9d5223dbddf1fea9b.